src/components/authentication/privateRoute.js

```diff
@@ -1,16 +1,15 @@
 import React, { Component } from 'react';
 import { Route, Redirect } from 'react-router-dom'
-
+import ApiService from '../customComponents/apiService';
 class PrivateRoute extends Component {
 
   render() {
     const logged = localStorage.getItem('loggedUser') ? true : false
+    const Comp = this.props.component;
     return (
       <Route render={props => (
-        logged ? React.createElement(
-          this.props.component,
-          {...this.props}
-        ) : (
+        logged ? ApiService.resetToken() && <Comp {...this.props} />
```

Comment:
shouldn't this be: `logged && ApiService.resetToken() ?`

Reply:
I think it could be either way, but the one you say seems more readable

Reply:
ended up removing this and doing differently